launcher_go/v2/cli_build.go

```diff
@@ -90,6 +90,51 @@ func (r *DockerConfigureCmd) Run(cli *Cli, ctx *context.Context) error {
 	return pups.Run()
 }
 
+type DockerMigrateCmd struct {
+	Config                       string `arg:"" name:"config" help:"config" predictor:"config"`
+	SkipPostDeploymentMigrations bool   `env:"SKIP_POST_DEPLOYMENT_MIGRATIONS" help:"Skip post-deployment migrations. Runs safe migrations only. Defers breaking-change migrations. Make sure you run post-deployment migrations after a full deploy is complete if you use this option."`
+}
+
+func (r *DockerMigrateCmd) Run(cli *Cli, ctx *context.Context) error {
+	config, err := config.LoadConfig(cli.ConfDir, r.Config, true, cli.TemplatesDir)
+	if err != nil {
+		return errors.New("YAML syntax error. Please check your containers/*.yml config files.")
+	}
+	containerId := "discourse-build-" + uuid.NewString()
```

Comment:
Let's use a fixed string when running the tests since it makes the `containerId` more deterministic hence more easier to test. Maybe extract what I did into a function?

https://github.com/discourse/discourse_docker/blob/7c18fce1b4cc766be1d3ce5d40edbc9cb8901491/launcher_go/v2/cli_build.go#L71-L79